extensions/arc/src/ui/components/radioOptionsGroup.ts

```diff
@@ -59,10 +64,12 @@ export class RadioOptionsGroup {
 				}));
 				this._divContainer.addItem(radioOption);
 			});
+			this.component().loadingCompletedText = this._loadingCompleteMessage;
 		}
 		catch (e) {
-			const errorLabel = this._modelBuilder.text().withProperties({ value: getErrorMessage(e), CSSStyles: { 'color': 'Red' } }).component();
+			const errorLabel = this._modelBuilder.text().withProperties({ value: `Error loading cluster contexts: ${getErrorMessage(e)}`, CSSStyles: { 'color': 'Red' } }).component();
```

Comment:
localize?

Reply:
Hah, I looked over this like 3 times too...